cpp/src/parquet/column_reader.h

```diff
@@ -28,6 +28,7 @@
 #include "parquet/properties.h"
 #include "parquet/schema.h"
 #include "parquet/types.h"
+#include "parquet/thrift_internal.h"  // IWYU pragma: keep
```

Comment:
This might fall the CI check due to exposing thrift symbols.

Reply:
Done.